<!-- -->
<launch>
    <node
            type="manager.py"
            pkg="adeye"
            name="manager"
            respawn="false"
            output="screen"
    />

    <node pkg="nmea_navsat_driver" type="nmea_serial_driver" name="navsat_driver" respawn="true">
        <param name="port" value="/dev/ttyACM0"/>
        <param name="baud" value="9600"/>
    </node>

    <node pkg="adeye" type="mocker.py" name="mocker" />

    <include file="$(find ouster_ros)/launch/driver.launch">
        <arg name="sensor_hostname" value="os-122226000469.local"/>
    </include>

    <!--
    <node
        type="launchDemo.sh"
        name="launchScript"
        pkg="adeye"
    />
    -->

    <node
        pkg="rostopic"
        type="rostopic"
        name="initial_checks_faker"
        args='pub /initial_checks std_msgs/Bool "data: true" -r 1'
        output="screen"
    />

    <node
        pkg="rostopic"
        type="rostopic"
        name="activation_request_faker"
        args='pub /activation_request std_msgs/Bool "data: true" -r 1'
        output="screen"
    />

 <!-- uncomment when  "Can't Generate Global Path for Start.." error is fixed-->
    <!-- <node
            type="sample_goal_publisher.py"
            pkg="adeye"
            name="goal_publisher"
            respawn="false"
            output="screen"
    />
-->

<!--    <node pkg="topic_tools" type="relay" name="points_raw_relay" output="log" args="/ouster/points /points_raw"/>  -->


</launch>
